src/cron.test.ts

```diff
@@ -2,7 +2,15 @@ import { getAllRFCRemarks } from "./cron";
 
 describe("RFC Listing test", () => {
   test("Should not return any remark with future date", async () => {
-    const remarks = await getAllRFCRemarks(new Date());
-    expect(remarks).toHaveLength(0);
-  }, 60_000);
+    const { ongoing, completed } = await getAllRFCRemarks(new Date());
+    expect(ongoing).toHaveLength(0);
+    expect(completed).toHaveLength(0);
+  }, 90_000);
+
+  test("Should return completed projects in the past", async () => {
```

Comment:
This test is using external api and it's gonna keep growing (with every new referenda on the chain).

So with every test run, we will keep hitting subsquare with requests about all referenda.

Would it make sense to add an optional `endDate` parameter, and use it here so we keep requesting a small, constant amount of referenda from subsquare?

Reply:
Actually, you gave me a great idea!

The approval object is something along the lines of `[ 1603062, null, null ]`.

That first number is the block time.

And it is the same time as the one I'm getting from the subsquare api. I can check that before hand!

Reply:
It works!

I'll be moving the subsquare api to a different method so we can properly test it with the tests